lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -617,6 +617,28 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     return nullptr;
   }
 
+  if (auto neScalar = dyn_cast<AtenNeScalarOp>(op)) {
+    Type dtype = neScalar.self().getType().cast<BaseTensorType>().getDtype();
+    Value otherPromoted =
+        convertScalarToDtype(b, loc, operands[1], payloadArgs[0].getType());
+
+    if (dtype.isa<mlir::FloatType>())
+      return b.create<arith::CmpFOp>(loc, arith::CmpFPredicate::UNE,
+                                     payloadArgs[0], otherPromoted);
+    if (dtype.isa<mlir::IntegerType>()) {
```

Comment:
Can you refer to https://github.com/llvm/torch-mlir/blob/5b9bdfaf3fb56c5aed12d689e21c9884b1c5a789/lib/Conversion/TorchToLinalg/Uncategorized.cpp#L153 and add something similar like `createNe`?

Reply:
Added PTAL.